src/pages/create.tsx

```diff
@@ -0,0 +1,183 @@
+            onChange={(e) => {
+              setProgramName(e.target.value);
+            }}
+          />
+        </div>
+        <div className="h-3" />
+
+        <Button
+          disabled={programName.length == 0}
+          onClick={() => {
+            if (programName) {
+              setStage((prev) => prev + 1);
+              setError("");
+            } else setError("Please enter a program name.");
+          }}
+        >
+          Next
+        </Button>
+        <div className="h-6" />
+
+        <Text className="text-error">{displayError}</Text>
+      </div>
+    );
+  };
+
+  const stepTwo = () => {
+    return (
+      <div className="flex flex-col w-full px-4 py-12 md:w-120">
+        <Text h1 b>
+          Program Details
+        </Text>
+        <Text>
+          Let's set up some basic information! You can also do these later.
+        </Text>
+        <div className="h-6" />
+        <div>
+          <Text b>Logo</Text>
+          <div className="h-1" />
+          <Button variant="inverted" size="small">
+            Choose File
+          </Button>
+          <input // TODO: display preview of logo
+            type="file"
+            name="Program Logo"
+            value={programLogo}
+            className="flex-1"
+            onChange={(e) => {
+              setProgramLogo(e.target.value);
+            }}
+          />
+        </div>
+
+        <div className="h-6" />
+        <Text b>Identifier</Text>
+        <div className="inline">
+          <Text b2>www.mentorcenter.us/</Text>
+          <Input
+            title="Identifier"
+            name="Program Identifier"
+            placeholder="nova-mentorship"
+            value={programIdentifier}
+            className="flex-1"
+            onChange={(e) => {
+              setProgramIdentifier(e.target.value);
+            }}
+          />
+        </div>
+
+        <div className="h-3" />
+        <div className="flex">
+          <Button variant="inverted">Back</Button>
+          <div className="w-2"></div>
+          <Button
+            disabled={programIdentifier.length == 0}
+            onClick={() => {
+              if (displayError || loading) {
+                // TODO: error, check identifier, bad file?
+              } else {
+                setLoading(true);
+                const createProgramInput: CreateProgramInput = {
+                  name: programName,
+                  description: "", // a lotta dummy strings for now since the form doesn't specify them
+                  slug: programIdentifier,
+                  iconUrl: programLogo,
+                  homepage: programIdentifier,
+                  mentorProfileSchemaJson: "",
+                  menteeProfileSchemaJson: "",
+                  mentorApplicationSchemaJson: "",
+                  public: true,
+                };
+                createProgram({ variables: { data: createProgramInput } }).then(
+                  () => setLoading(false)
+                );
```

Comment:
Adding a .catch statement to catch any errors that may return from calling this function may be helpful. Refer to:
```
createProgram({...})
.then(() => {...})
.catch((error) => { setError(error.message); })
```

Reply:
As context, I entered a duplicate identifier and it resulted in this error:
![image](https://user-images.githubusercontent.com/20727715/120911924-e9a3d680-c63f-11eb-8b9b-9d745a37c5f7.png)
If we catch the error, this could be integrated into the UI instead 👍

Reply:
u so smart ty